datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -314,6 +314,14 @@ def metadata(check, check_duplicates):
             # empty description field, description is recommended
             if not row['description']:
                 empty_warning_count['description'] += 1
+
+            # see if there is unicode in the description
+            elif not row['description'].isascii() and row['metric_name'].isascii() and row['metric_type'].isascii():
+                errors = True
+                echo_failure(
+                    f"{current_check}:{line} `{row['metric_name']}`\'s description contains unicode characters."
```

Comment:
Since it's not just the description, let's just give the line number

Reply:
Sure, I forgot to update it!